packages/sdk-react/src/components/approvalPolicies/ApprovalPolicyDetails/ApprovalPolicyDetails.tsx

```diff
@@ -39,11 +39,11 @@ export const ApprovalPolicyDetails = ({
 
   const {
     isLoading,
-    isInitialLoading,
+    isPending,
     data: approvalPolicy,
   } = useApprovalPolicyById(id);
 
-  if (id && (isLoading || isInitialLoading)) {
+  if (id && isLoading && isPending) {
```

Comment:
Probably, `if (id && isLoading)` would be enough for the case.

Reply:
left only `isLoading ` here